doc/hardware/porting/board_porting.rst

```diff
@@ -158,6 +166,10 @@ The mandatory files are:
    software configuration in :ref:`kconfig` formats. This provides default
    settings for software features and peripheral drivers.
 
+.. note::
+
+	For **downstream** boards, the only mandatory file is the _defconfig file.
+
```

Comment:
no, this is still not correct.

If you follow the board porting guide, then the devicetree file is mandatory.
As well as if you use and architecture natively supported by Zephyr, such as arm, riscv, etc.

But if you're implementing your own arch, soc, and board, then you are able to not use dt under some very strict conditions. Therefore, this is really advanced users, and not something to describe in board porting guide (which targets users needing to create a new board based on an existing SoC in Zephyr (and thus is based on an existing arch).

For a board porting guide that describes how to add your own board (using an arch defined in Zephyr), then the dts file is mandatory.

From the build system perspective though, things are a little more complex, hence the reason I mentioned why the dts file cannot be used to determine if a board is valid.

```suggestion
```

Reply:
okay, I will just delete this part. It wasn't the main purpose of this PR